providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/InProcessResolver.java

```diff
@@ -46,9 +46,10 @@ public class InProcessResolver implements Resolver {
      * Initialize an in-process resolver.
      */
     public InProcessResolver(FlagdOptions options, Consumer<ProviderState> stateConsumer) {
-        final Connector connector = options.isOffline()
-                ? new FileConnector(options.getOfflineFlagSourcePath())
-                : new GrpcStreamConnector(options);
+        final Connector connector =
```

Comment:
maybe we could add some tests in InProcessResolverTest.java to check that the right connector is selected but def not needed as part of this PR :)

Reply:
Done, added a test with commit https://github.com/open-feature/java-sdk-contrib/pull/647/commits/f2c331338e96047723f49d3cea9e809301e1d6d4